src/parse/ast.ts

```diff
@@ -37,6 +37,10 @@ export const parseNameFromNode = (node: RawNode): string => {
       names.push(parseNameFromNode(child));
     });
     return names.join(",");
+  } else if (ts.isIdentifier(node)) {
+    if (node.escapedText && typeof node.escapedText === "string") {
```

Comment:
This is also a part I'm not super sure about personally. Maybe it would help if you had a comment in case this has an issue and needs to be fixed later?

Reply:
Ok, let me add a comment